beacon-chain/blockchain/process_block.go

```diff
@@ -181,7 +181,12 @@ func (s *Service) onBlock(ctx context.Context, signed interfaces.SignedBeaconBlo
 		return errNilJustifiedInStore
 	}
 	currJustifiedEpoch := justified.Epoch
-	if postState.CurrentJustifiedCheckpoint().Epoch > currJustifiedEpoch {
+	psj := postState.CurrentJustifiedCheckpoint()
+	if psj == nil {
```

Comment:
I know it's not part of the PR but we could please encapsulate nil checkpoint error into `CurrentJustifiedCheckpoint() error`?

I'll be happy to do this in another PR

Reply:
I'm doing it here: https://github.com/prysmaticlabs/prysm/pull/10729